packages/webcrypto-ed25519-polyfill/src/__tests__/secrets-test.ts

```diff
@@ -44,6 +84,161 @@ describe('exportKeyPolyfill', () => {
     });
 });
 
+describe('importKeyPolyfill', () => {
+    describe('when format is `raw`', () => {
+        it('allows importing valid public key bytes', () => {
+            expect(() => importKeyPolyfill('raw', MOCK_PUBLIC_KEY_BYTES, false, ['verify'])).not.toThrow();
+        });
+        it('allows importing with empty keyUsages', () => {
+            expect(() => importKeyPolyfill('raw', MOCK_PUBLIC_KEY_BYTES, false, [])).not.toThrow();
+        });
```

Comment:
[![orlyowl](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lE1zD7Jpg6mWNv3djNKZ/6a784326-98ec-404c-94e8-07f823629513.gif)](https://app.graphite.dev/macros/solana-labs)

Reply:
If someone imports with no usages, does this code still actually allow them to use it with `verify`? It shouldn't, if so.

Reply:
Yup! I was doing a pass to make sure my error messages all matched the real ones and realised this didn't actually error. I guess it makes sense because you could generate a keypair with just `["verify"]` and then if you exported and re-imported that private key it'd have empty usages.